cas-config/models.py

```diff
@@ -0,0 +1,22 @@
+from django.conf import settings
```

Comment:
* It's odd to me that the name of this module is not a valid Python module name. (since there's a dash in the file name)
* It's also odd to me that this is in `models.py`, which by Django app expectations would contain database models. Could this be named `receivers.py`?
* If this is an optional part, could this live under `contrib` to avoid clutter?

Reply:
> It's odd to me that the name of this module is not a valid Python module name. (since there's a dash in the file name)

I was foolishly imitating `saml-config`. Which isn't a module, come to think of it. Renamed to `cas_config` unless you can offer a better name.

> It's also odd to me that this is in models.py, which by Django app expectations would contain database models. Could this be named receivers.py?

Sorry, force of habit (we still have yet to migrate to Django 2.0). In 2.0, this can be attached by apps.py and named whatever...

> If this is an optional part, could this live under contrib to avoid clutter?

This isn't really an optional part, because it's required for the below-described `callback.py` to work if it is present. An example callback.py could be in contrib, though, which is where I'm going to put it...